web/package.json

```diff
@@ -116,5 +117,11 @@
       "src/serviceWorker.ts",
       "src/app/store.ts"
     ]
+  },
+  "overrides": {
+    "typescript": "^5.0.0"
```

Comment:
Why do we need to override typescript?

Reply:
This is my only question as well, out of curiosity

Reply:
Good question! Originally it looked like we were going to have to install our packages with npm instead of yarn in order to get pmtiles working and this override was required.
<img width="672" alt="image" src="https://github.com/bcgov/wps/assets/7094320/2612f704-e6b2-4083-9ae5-815ca179b35c">

Fortuantely we can continue using yarn and this override will no longer be required.